bioptim/dynamics/dynamics_type.py

```diff
@@ -12,16 +12,25 @@ class DynamicsFcn(Enum):
 
     TORQUE_DRIVEN = (Problem.torque_driven,)
     TORQUE_DRIVEN_WITH_CONTACT = (Problem.torque_driven_with_contact,)
+
+    TORQUE_DERIVATIVE_DRIVEN = (Problem.torque_derivative_driven,)
+    TORQUE_DERIVATIVE_DRIVEN_WITH_CONTACT = (Problem.torque_derivative_driven_with_contact,)
+
     TORQUE_ACTIVATIONS_DRIVEN = (Problem.torque_activations_driven,)
     TORQUE_ACTIVATIONS_DRIVEN_WITH_CONTACT = (Problem.torque_activations_driven_with_contact,)
 
     MUSCLE_ACTIVATIONS_DRIVEN = (Problem.muscle_activations_driven,)
     MUSCLE_ACTIVATIONS_AND_TORQUE_DRIVEN = (Problem.muscle_activations_and_torque_driven,)
     MUSCLE_ACTIVATIONS_AND_TORQUE_DRIVEN_WITH_CONTACT = (Problem.muscle_activations_and_torque_driven_with_contact,)
 
+    # MUSCLE_ACTIVATIONS_AND_TORQUE_DERIVATIVE_DRIVEN = (Problem.muscle_activations_and_torque_derivative_driven,) TODO
```

Comment:
Please use `# TODO` (as opposed to not having the `#`) as this will be highlighted properly by pycharm